docs/source/ephys_generator.md

```diff
@@ -55,7 +55,11 @@ To start the `ephys generator`, complete the [installation](installation.md), ad
 ephys_generator
 ```
 
-The script can use a different config file specified via the `--settings-path` argument.
+To use a specific config file, specify the config directory (`--config-dir`) and config file-name (`--config-name`) flags. For example:
+
+```
+ephys_generator --config-dir $HOME/.nds/ --config-name settings
```

Comment:
have you tried using a --config-dir different from the default NDS_HOME path (os.path.join(os.path.expanduser("~"), ".nds"))? 
I think it won't work as the runtime.py has the `NDS_HOME` variable that is used in the hydra decorators but it's not set by the user input

Reply:
Thanks for catching that.

Looks like `--config-path` is the correct way to overwrite the config, whereas `--config-dir` is only appends to the config-search-path. So something like:
```
streamer --config-path=/Users/charles/Development/nds/src/streamer/config/
```
still works after removing my `~/.nds/*.yaml` configs.

Unfortunately, it requires an absolute path (or a path-relative to `streamer`, which is probably a bit confusing for most users).

Let me see if there is a way to also pass in a config path relative to the current working directory.